pallets/phala/src/registry.rs

```diff
@@ -586,14 +589,14 @@ pub mod pallet {
 		pub fn register_worker_v2(
 			origin: OriginFor<T>,
 			pruntime_info: WorkerRegistrationInfoV2<T::AccountId>,
-			attestation: Option<AttestationReport>,
+			attestation: Box<Option<AttestationReport>>,
```

Comment:
`Option<Box<AttestationReport>>` might be better, since it allows passing `None` rather than `Box::new(None)`.

Reply:
Fixed